modules/lib/lib-node/src/main/resources/lib/xp/node.ts

```diff
@@ -641,7 +637,7 @@ export interface QueryNodeParams {
     sort?: string | SortDsl | SortDsl[];
     filters?: Filter | Filter[];
     aggregations?: Record<string, Aggregation>;
-    suggestions?: Record<string, Suggestion>;
+    suggestions?: Record<string, TermSuggestion>;
```

Comment:
The reason why I removed `Suggestion` type and placed `TermSuggestion` here was that it was basically an alias for that. Having two same types with different names exported doesn't look very wise. Since we haven't released any stable types yet, it'll be better to export as less as possible if we are not sure how this types will look in future releases. It's always easier to add types, than remove them, because in the second case it can break stuff.